### PR TITLE
chore!: fix cancel_deferred_save event typo

### PR DESCRIPTION
**Pull request by primeapple**
Copied over from https://github.com/okuuva/auto-save.nvim/pull/48

### CODE REVIEW

**Comment by primeapple** [comment]
@okuuva I tested it locally and it seems to work. However the warning message only shows up for a split second.
Could you test this yourself? Maybe it's just a problem with my config.

**Comment by okuuva** [comment]
> @okuuva I tested it locally and it seems to work. However the warning message only shows up for a split second. Could you test this yourself? Maybe it's just a problem with my config.

Hey, sorry for the late reply. I've been both away from home and sick for the past week or so :/ Still not quite 100% but I'll try to check it tomorrow once I get back home

File: lua/auto-save/config.lua [hunk 1]
```diff
@@ -34,13 +34,22 @@ function Config:handle_deprecations(custom_opts)
     custom_opts["execution_message"] = nil
   end
 
+  if custom_opts["cancel_defered_save"] then
```
Comment (okuuva) [inline lua/auto-save/config.lua:37]: `cancel_defered_save` isn't a top level setting. Should this check for `custom_opts["trigger_events"]["cancel_defered_save"]` instead?

Comment (okuuva) [reply lua/auto-save/config.lua:37]: I just tested this. It does not react to `cancel_defered_save` inside `trigger_events` as it should but does react to a top-level `cancel_defered_save` so the table level isn't right.

Comment (primeapple) [reply lua/auto-save/config.lua]: Ohh, my bad. I fixed it.

**Comment by primeapple** [comment]
Ok, everything fine now. If you approve I'll merge :)

**Review by okuuva — APPROVED** [review]
LGTM!

**Pull request merged**